src/views/Home.jsx

```diff
@@ -90,33 +93,58 @@ export function Home({ setListToken }) {
 	};
 
 	return (
-		<div className="Home">
-			<p>
-				Hello from the home (<code>/</code>) page!
+		<div className="home">
+			<p>Welcome to</p>
+			<h1>App Title!</h1>
```

Comment:
I'm not sure how much styling you did in this issue, so maybe disregard, but "Welcome to" is a different color and font from the other text font below in `<p>` (on second look I think the text below may be in the wrong font because some is serif and some is sans-serif, and I think we decided that non-headers are sans-serif? Also, it looks like the h1 font is a different color from the rest of the text.

Reply:
@sarahmjenkins Ah yea, for the colors and fonts I was following the wireframe I had. Things might have changed in the wireframe the last time I looked, though. Gonna have to take a look back at Figma!